docs/azure-devops.md

```diff
@@ -153,19 +149,11 @@ The output consists of the following:
 - An overview of all ratings, compared against the system as a whole. This allows you to check if your changes improved the system, or accidentally made things worse.
 - The final conclusion on whether your changes and merge request meet the quality target.
 
-In addition to the textual output, Sigrid CI also generates a static HTML file that shows the results in a more graphical form. This is similar to test coverage tools, which also tend to produce a HTML report. You can access the HTML report from the "published" section in the build summary.
+In addition to the textual output, the results also include a link to the Sigrid landing page. 
 
-<img src="images/azure-artifacts.png" width="500" />
+<img src="images/landing-page.png" width="700" />
 
-In the list of published artifacts, expand the "sigrid-ci-output" section and download the index.html file to view the report.
-
-<img src="images/azure-artifact-download.png" width="600" />
-
-The information in the HTML report is similar to the command line output, though it includes slightly more detail.
-
-<img src="images/feedback-report.png" width="600" />
-
-Finally, if you want to have more information on the system as a whole, you can also access [Sigrid](http://sigrid-says.com/), which gives you more information on the overall quality of the system, its architecture, and more.
+You can open this URL in order to use Sigrid for interpreting your analysis results. Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

Comment:
The link to the Sigrid landing page is technically part of the textual output (contrary to what was mentioned in the old text). I wanted to suggest adding it to the list as another bullet but I think it's clearer to indeed have this text separated from the list. I would however change the text accordingly. I would also move the first sentence of the next paragraph here. The result would be the following:

```
At the end of the textual output there is a link to the Sigrid landing page. You can open this URL in order to use Sigrid for interpreting your analysis results.

<img src="images/landing-page.png" width="700" />

Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

(I can't seem to put this as a suggestion directly as GitHub won't allow those on removed lines and even in split mode I can't only select the new and unchanged lines...)